ui/src/tanagra-underlay/underlayConfig.ts

```diff
@@ -26,7 +26,7 @@ export enum SZCorePlugin {
   OUTPUT_UNFILTERED = "OUTPUT_UNFILTERED",
   TEXT_SEARCH = "TEXT_SEARCH",
   UNHINTED_VALUE = "UNHINTED_VALUE",
-};
+}
```

Comment:
Sorry, I was telling you to put the fix in the wrong place. It should go in `ui/.eslintignore`. Putting it here won't work because this is generated.

Reply:
Is this generated? Looks like Mariko made changes here last week: https://github.com/DataBiosphere/tanagra/pull/820/files#diff-bf1a95422b29fc148e786e4b143abb98804528f2181280ab2f825a476fc46ea3

Reply:
This file is generated as part of the backend build which is why it's checked in so it doesn't need to be run separately before working on the UI: `./gradlew annotationProcessor:run -Pgenerator=UNDERLAY_CONFIG_TYPESCRIPT`. This is different from the the the UI specific stuff generated by `npm run codegen`, which is what would have made sense to go in .gitignore.

Reply:
Ah, makes sense. Thanks